.github/workflows/test.yaml

```diff
@@ -177,8 +177,8 @@ jobs:
           # Start a TryCP instance
           nix develop .#ci -c bash -c "source ./scripts/trycp.sh && start_trycp &"
 
-          # Run the scenario for 10 seconds
-          RUST_LOG=warn CONDUCTOR_CONFIG="CI" MIN_PEERS=2 nix run .#trycp_write_validated -- --targets targets-ci.yaml --instances-per-target 2 --duration 10 --no-progress
+          # Run the scenario for 30 seconds
+          RUST_LOG=warn CONDUCTOR_CONFIG="CI" MIN_PEERS=2 nix run .#trycp_write_validated -- --targets targets-ci.yaml --instances-per-target 2 --duration 30 --no-progress
```

Comment:
Is it these timings that have the test time go from [12m](https://github.com/holochain/wind-tunnel/actions/runs/11498921230/job/32005758024) to [43m](https://github.com/holochain/wind-tunnel/actions/runs/11501744059/job/32015030249?pr=115)?

And if so, are we okay with that?

Reply:
I'm wondering what we've been running as tests. The smoke tests on a recent workflow run all output the error saying that the app couldn't be installed, like here https://github.com/holochain/wind-tunnel/actions/runs/11498921230/job/32005758024#step:24:87

Is that a false error message?

Reply:
@neonphog, I don't think so / hope not. There are only 5 tests that were changed from 10 to 30 seconds so I'd expect an increase of 100 seconds. I think that the large difference here is just because it was the first build and so there's no cache as even the Rust Clippy job went from ~2 minutes to 12.5 minutes.

Reply:
@jost-s, from what I saw running locally and with CI, 10 seconds is not enough time to even install the apps so I think that the error message is correct.

Reply:
Sounds right to me, some of the scenarios need more time to get going, so fair enough if they need more time. I roughly checked using logs that I'd given about enough time but hadn't got this far yet. The scenario is absolutely meant to run at least one behavior loop. 

Maybe this would be better drive with a proper test harness at some point rather than the real runner, where we can guarantee a limited number of passes doesn't produce errors... but even that is tricky because the behavior is allowed to error :)

Reply:
If the app installation fails for a percentage of agents, should the test fail too? In the workflow run they're shown as successful.

Reply:
The agent is meant to shut down if they run into a fatal error. So if the errors aren't fatal then they should get recorded.

That does still leave room for CI missing a breakage I know, but I don't want too much in the framework that is specific to CI if we can help it. The behavior functions need to be allowed to fail, because they validly do sometimes

Reply:
Open to suggestions like this PR though, if we can make CI work better without adding too much to Wind Tunnel that might accidentally cause problems in real runs

Reply:
Is there another explanation of the vastly increased test run duration?

Reply:
I don't think so, I believe the answer @cdunster gave is accurate. The CI relies quite a bit on caching, if you make changes that require everything to be rebuilt then it takes quite a bit longer. Changing a single scenario is fairly quick to build but changing the framework requires all scenarios to be rebuilt and takes longer

It might be that we can make the CI faster but I don't know by how much